<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowparticipants</class>
 <widget class="QDialog" name="windowparticipants">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Des Participants</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 127)</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>761</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 127)</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>ajouter un participant</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>731</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="title">
      <string>Formulaire Ajout Participant</string>
     </property>
     <widget class="QPushButton" name="pushbotton_ajouter">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>100</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>160</width>
        <height>180</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLineEdit" name="le_nom">
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_id">
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_Pays">
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
         </property>
         <item>
          <property name="text">
           <string>Pays</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tunisie</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Canada</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>USA</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_cin">
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>189</x>
        <y>40</y>
        <width>71</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>32</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Pays</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cin</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>afficher un participant</string>
    </attribute>
    <widget class="QTableView" name="tab_participant">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>541</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>Tri</string>
     </property>
    </widget>
    <widget class="QPushButton" name="imprimerListP">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;
font-size:11px;</string>
     </property>
     <property name="text">
      <string>Imrpimer Tous</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Stat_p">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>Statistique</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>traduction</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nom_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>recherche</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>83</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
     </property>
     <property name="text">
      <string>Tri</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>supprimer participant</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_modifier_id">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>106</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>modifier participant</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>120</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>160</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="styleSheet">
         <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="pays_modifier">
        <property name="styleSheet">
         <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
        </property>
        <item>
         <property name="text">
          <string>Pays</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tunisie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Canada</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USA</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="styleSheet">
         <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>160</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Pays</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cin</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Imprimer 1 Seul</string>
    </attribute>
    <widget class="QComboBox" name="combo_id_p">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID PARTICIPANT A IMPRIMET</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Pdf">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>260</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>490</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Retour au menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
